apps/examples/src/examples.tsx

```diff
@@ -7,22 +7,99 @@ export type Example = {
 	path: string
 	codeUrl: string
 	hide: boolean
+	category: 'basic' | 'editor' | 'ui' | 'collaboration' | 'data/assets' | 'shapes/tools' | null
 	order: number | null
 	componentFile: string
 	loadComponent: () => Promise<ComponentType>
 }
 
-export const examples = (
+const basicExamples = (
```

Comment:
so, for this logic below, what would be better is to consolidate it (make it so when adding a category/removing a category, you don't have to think of another place where you have to add a codeblock/maintain extra code to make something work)

we should make it more like:
```
enum CATEGORIES = {
  BASIC = 'basic',
  EDITOR = 'editor'
  etc...
}

const CATEGORY_TO_TITLE = {
  BASIC = 'Getting Started',
  EDITOR = 'Editor API'
  etc...
}

export const examples = CATEGORIES.map(category => {
  Object.values(import.meta.glob('./examples/*/README.md', { eager: true })) as Example[]
)
	.filter((e) => e.category === category)
  ...

  return { id: CATEGORY_TO_TITLE[category], values: ...}
})
```
(also, above in the type, you can do something like: `category: CATEGORIES` instead of listing it out again)